packages/memoize/src/index.ts

```diff
@@ -1,7 +1,9 @@
-export default function memoize<V>(fn: (arg: string) => V): (arg: string) => V {
-  const cache: Record<string, V> = Object.create(null)
+export default function memoize<K extends PropertyKey, V>(
```

Comment:
In a way this could be seen as a breaking change - I agree with the change though. (we don't actually need it but `@emotion/memoize` is a pretty generic helper, so from its perspective the adjusted declaration is better).

To maintain compatibility we could introduce the old signature as an overload (it should come after the existing one).

Reply:
You are right, I was not even thinking that this is a breaking change...